zebra-rpc/src/errors.rs

```diff
@@ -1,5 +1,9 @@
 use jsonrpc_core::types::error::{Error, ErrorCode};
```

Comment:
Please add a module doc comment to new modules.

Reply:
I removed `make_server_error` for now and created #5548 for the refactor suggested above.